styles/about.css

```diff
@@ -0,0 +1,118 @@
+html, body {
```

Comment:
There's a lot of repetition among the style sheets. Consider moving the common stuff into a `site.css` sheet that can be included by all the pages, and then just put the page overrides in the per-page files.